path_config.go

```diff
@@ -110,9 +117,9 @@ func (b *azureAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Req
 		config.Resource = resource.(string)
 	}
 
-	environment, ok := data.GetOk("environment")
+	cloud_config, ok := data.GetOk("cloud_config")
```

Comment:
Suggest to use camelCase for variables. Though I think we'll want to bring the environment stuff back and then use the cloud config internally.

Reply:
Reverted to `environment` in latest commit